### PR TITLE
Add necessary infrastructure to be able to use travis CI with this repository

### PR DESCRIPTION
**Pull request by ZachEichen**
Adds a .travis.yaml file that sets up an environment for and runs unit regression tests.  Addresses issue #33

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+ - source "$HOME/miniconda/etc/profile.d/conda.sh"
+ - hash -r
+ - conda config --set always_yes yes --set changeps1 no
+ - conda update -q conda
+
+ - conda info -a
+
+ - conda create --name pd
+ - conda activate pd
+ - conda install tensorflow jupyterlab pandas regex matplotlib cython grpcio-tools pytorch scikit-learn
```
Comment (frreiss) [inline .travis.yml:19]: Can you move this list of packages and the one on the next line to an `environment.yml` file, and update `env.sh` to read package names the same `environment.yml` file?

Comment (ZachEichen) [reply .travis.yml]: Should be fixed by my latest commit

**Comment by ZachEichen** [comment]
I did check the travis CI on my own personal fork.

**Comment by frreiss** [comment]
Latest changes look good. In other news, Travis CI saw that this PR has a `.travis.yml` file and attempted to run tests. Tests almost worked, but there are some strange failures probably due to differences in the versions of SpaCy and its models that get installed with the new configs. I'm going to merge these changes into master so we'll have a basis to do some follow-on PRs to clean up the environment and build scripts.

**Pull request merged**